build.sbt

```diff
@@ -23,7 +23,8 @@ val `interpolators` = project
     name := "mobilityid-interpolators",
     commonSettings,
     libraryDependencies ++= Seq(
-      "com.propensive" %% "contextual" % "1.1.0",
+      "com.propensive" %% "contextual-core" % "3.0.0",
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value,
```

Comment:
do we need this? it seems to work for me without it

Reply:
It was needed for 2.0.0, but 3.0.0 no longer. I removed it